src/transport.rs

```diff
@@ -85,20 +84,21 @@ impl Transport {
     pub fn new(dsn: Dsn, user_agent: String) -> Transport {
         let (sender, receiver) = sync_channel(30);
         let drain_signal = Arc::new(Condvar::new());
+        #[cfg_attr(feature = "cargo-clippy", allow(mutex_atomic))]
         let queue_size = Arc::new(Mutex::new(0));
```

Comment:
Did not change this `Mutex`, as I'm pretty sure we want to block around `signal.notify_all()`